app/services/albumsManager.js

```diff
@@ -9,10 +9,17 @@ exports.listAlbums = () => {
   return new Promise(function(resolve, reject) {
     request(options, (error, response, body) => {
       if (error) reject(error);
-      else {
-        const albums = body.map(key => ({ id: key.id, title: key.title }));
-        resolve(albums);
-      }
+      const albums = body.map(key => ({ id: key.id, title: key.title }));
+      resolve(albums);
+    });
+  });
+};
+exports.findAlbum = albumId => {
+  return new Promise(function(resolve, reject) {
```

Comment:
repeating logic here with the promise and the request, you should make a wrapper here